src/ui/public/agg_response/tabify/_response_writer.js

```diff
@@ -24,23 +24,27 @@ import { tabifyGetColumns } from './_get_columns';
  * Writer class that collects information about an aggregation response and
  * produces a table, or a series of tables.
  *
- * @param {Vis} vis - the vis object to which the aggregation response correlates
+ * @param {AggConfigs} aggs - the agg configs object to which the aggregation response correlates
+ * @param {boolean} minimalColumns - setting to false will produce metrics for every bucket
```

Comment:
I think this needs adjustements to the new parameters `partialRows` and `metricsAtAllLevels` now.